src/controller_examples/rabbitmq_controller/spec/resource/mod.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: MIT
+pub mod common;
+pub mod config_map;
+pub mod default_user_secret;
+pub mod erlang_cookie;
+pub mod headless_service;
+pub mod rabbitmq_plugins;
+pub mod role;
+pub mod role_binding;
+pub mod service;
+pub mod service_account;
+pub mod stateful_set;
+
```

Comment:
Perhaps create a prelude.rs and move the `pub use *` below to prelude.rs

Reply:
Do you learn this from verus or this is a better practice?

Reply:
Learned from Verus. But I don't think it would affect much so it's ok to leave it as it is.

Reply:
Ok. I learn this from the k8s openapi :-))